src/main/java/com/oramon/kata/GameOfLife.java

```diff
@@ -0,0 +1,29 @@
+package com.oramon.kata;
+
+import com.oramon.kata.draw.DrawStrategy;
+import com.oramon.kata.draw.impl.DrawStrategyImpl;
+
+/**
+ * ClassName
+ * <p>
+ * Created by Alfredo Gil Rubio
+ * <p>
+ * Date: 26/05/17 11:57
+ */
+public class GameOfLife {
+    private int[][] world;
```

Comment:
A matrix can have too specific functionalities for your context, it's better than you encapsulate it. Because the gameoflife could have more functionalities and therefore it should be the manager of the all cases of the matrix